configuration/structures.go

```diff
@@ -164,12 +164,17 @@ type AgentFields struct {
 }
 
 // WorkScheduler represents work schedule data
-type WorkScheduler map[Weekday]WorkSchedulerDay
+type WorkScheduler struct {
+	Timezone string     `json:"timezone"`
+	Schedule []Schedule `json:"schedule"`
+}
 
-// WorkSchedulerDay represents single day work schedule
-type WorkSchedulerDay struct {
-	Start string `json:"start"`
-	End   string `json:"end"`
+// Schedule represent a single day work schedule
+type Schedule struct {
+	Enabled bool    `json:"enabled"`
```

Comment:
I think it's optional param but I am not sure. Maybe `*bool`? Look at [that line](https://github.com/livechat/api/blob/70b63fa2f01c85adf4edc783341883e5c16c427b/services/configuration-api/parsers/v3.5/objects.go#L32).

Reply:
Changed to `*bool`. Enabled is a required param. Tried making the call without it which resulted in:
```
{
    "error": {
        "type": "validation",
        "message": "`work_scheduler.schedule.enabled` is required"
    }
}
```

Reply:
I left `bool` as it was. With `*bool` the tests were failing.  `handler_test.go:166: Payload incorrectly parsed for agent_updated, error: webhook handler error: WorkScheduler.Schedule[0].Enabled mismatch, actual: 0x1400023a4e6, expected: true`

For it to work, I'd need to change [this line](https://github.com/livechat/lc-sdk-go/blob/4fcc7ff3df04f2a8d64b9ec9b26d3ded91046719/webhooks/helpers_test.go#L405) to expect a pointer, which wouldn't make sense, I think?